libc-test/build.rs

```diff
@@ -1466,7 +1467,10 @@ fn test_freebsd(target: &str) {
     // Required for `getline`:
     cfg.define("_WITH_GETLINE", None);
     // Required for making freebsd11_stat available in the headers
-    cfg.define("_WANT_FREEBSD11_STAT", None);
+    match freebsd_ver {
+        Some(10) => &mut cfg,
+        _ => cfg.define("_WANT_FREEBSD11_STAT", None),
```

Comment:
You shouldn't need this symbol . This is only needed on 12+ to expose the backwards-compatible definition of `struct stat` under a different name, and libc doesn't appear to be doing anything with it.